elements/howto-tabs/demo.html

```diff
@@ -11,40 +11,24 @@
 limitations under the License.
 -->
 <style>
-  howto-tabs {
-    /**
-     * The element uses flex box to line up the tabs in the first line
-     * and wraps if necessary.
-     */
-    display: flex;
-    flex-wrap: wrap;
-  }
   howto-tabs-tab {
```

Comment:
Those are just the embellishments. The element could provide default styles, I guess, but I thought I’d only inline the functional styles (and also these need to be here when JS is not working).